core/src/test/java/org/apache/iceberg/TestMetrics.java

```diff
@@ -270,6 +289,97 @@ public void testMetricsForNullColumns() throws IOException {
     assertBounds(1, IntegerType.get(), null, null, metrics);
   }
 
+  @Test
+  public void testMetricsForTopLevelWithMultipleRG() throws Exception {
+    Schema schema = getSimpleSchema();
+    int minimumRowGroupRecordCount = 100;
```

Comment:
Same as below. This isn't used, but 201 is used as a constant in lots of places.

Reply:
updated.